platforms/core-execution/persistent-cache/src/main/java/org/gradle/cache/CleanupFrequency.java

```diff
@@ -16,28 +16,27 @@
 
 package org.gradle.cache;
 
-import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
+import java.time.Duration;
+import java.time.Instant;
 
 /**
  * Represents when cache cleanup should be triggered.
  *
  * @since 8.0
  */
```

Comment:
❓ I wonder why this has `@since` given that the package is not part of the public API... 🤔